firmware/v5-common.ld

```diff
@@ -228,7 +226,7 @@ _SDA2_BASE_ = __sdata2_start + ((__sbss2_end - __sdata2_start) / 2 );
    _heap_end = .;
    HeapLimit = .;
 } > HEAP
-
+/*
```

Comment:
Surely commenting out this section without an explanation is not the best way to achieve this?

Reply:
I've added an explanation in https://github.com/purduesigbots/pros/pull/708/commits/8b9cfae4726ca3191efb34155bf8d3d13538a26c.